### PR TITLE
Create hazelcast-cli.rb

### PR DESCRIPTION
**Pull request by bilalyasar**

### CODE REVIEW

File: Library/Formula/hazelcast-cli.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                /usr/local/Library/Contributions/example-formula.rb
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```
Comment (bfontaine) [inline Library/Formula/hazelcast-cli.rb]: Please remove these comments.

File: Library/Formula/hazelcast-cli.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+class HazelcastCli < Formula
+  homepage "github.com/hazelcast"
```
Comment (bfontaine) [inline Library/Formula/hazelcast-cli.rb]: Please use 2-spaces indentation.

File: Library/Formula/hazelcast-cli.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  url "http://download.hazelcast.com/enterprise/brew/hazelcast-cli-1.0.tar.gz"
+  version "1.0"
```
Comment (bfontaine) [inline Library/Formula/hazelcast-cli.rb]: You can remove this, Homebrew extracts the version from the URL.

File: Library/Formula/hazelcast-cli.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "086896eb5d88daefd262fab31a7b81252cb2b04923b8053a6b455b9a3872ea0b"
+
+  # depends_on "cmake" => :build
```
Comment (bfontaine) [inline Library/Formula/hazelcast-cli.rb]: If this dependency is not needed you can remove it.

File: Library/Formula/hazelcast-cli.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+  def install
+   libexec.install Dir["*"]
+    bin.install_symlink "#{libexec}/hazelcast.sh" => "hazelcast"
+  end
+
+  test do
+    system "false"
```
Comment (bfontaine) [inline Library/Formula/hazelcast-cli.rb]: Could you please add a test for `hazelcast`? Thanks!

File: Library/Formula/hazelcast-cli.rb [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                /usr/local/Library/Contributions/example-formula.rb
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class HazelcastCli < Formula
```
Comment (bfontaine) [inline Library/Formula/hazelcast-cli.rb]: Could you add a short description of the formula using `desc "…"`? Thanks!

File: Library/Formula/hazelcast-cli.rb [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "github.com/hazelcast"
+  url "http://download.hazelcast.com/enterprise/brew/hazelcast-cli-1.0.tar.gz"
+  version "1.0"
+  sha256 "dbe914190907c6c91adf0d9e0262e8164f59adb92e81e6efbb0e88134dea4275"
+
+  # depends_on "cmake" => :build
+
+  def install
+   libexec.install Dir["*"]
```
Comment (mikemcquaid) [inline Library/Formula/hazelcast-cli.rb]: Fix the indentation here.

**Comment by DomT4** [comment]
Ping. Lots of comments above that need looking over.

**Comment by mozinrat** [comment]
When can this be merged, thank you

**Comment by DomT4** [comment]
When the issues are fixed, essentially. The PR hasn't seen any activity since submission, so if someone wants to pick it up and run with it that's fine.

**Comment by bilalyasar** [comment]
i will take care of this PR.

**Comment by DomT4** [comment]
Ping?

**Comment by DomT4** [comment]
Closing as dead. Feel free to address the comments resubmit this fresh if still interested. Thanks for the submission!

**Pull request closed without merging**